encrypt.go

```diff
@@ -180,6 +188,60 @@ func encryptDESCBC(content []byte, key []byte) ([]byte, *encryptedContentInfo, e
 	return key, &eci, nil
 }
 
+func encryptAESCBC(content []byte, key []byte) ([]byte, *encryptedContentInfo, error) {
+	var keyLen int
+	var algID asn1.ObjectIdentifier
+	if ContentEncryptionAlgorithm == EncryptionAlgorithmAES128CBC {
```

Comment:
maybe a `switch ContentEncryptionAlgorithm` would be better? That was the default case could return an error.

Reply:
Should I make this change to `encryptAESGCM` as well?

Reply:
that might be a good idea

Reply:
Done in https://github.com/mozilla-services/pkcs7/pull/17/commits/a91be25d3f912154e8cad900c0acab4e3dba20df